src/components/views/EditDAC.jsx

```diff
@@ -259,10 +259,10 @@ class EditDAC extends Component {
                     <Input
                       name="title"
                       id="title-input"
-                      label="Community cause"
+                      label="Name your Community"
                       type="text"
                       value={dac.title}
-                      placeholder="e.g. Hurricane relief."
+                      placeholder="Describe your altruistic community (DAC) in 1 sentence"
                       help="Describe your Decentralized Altruistic Community (DAC) in 1 sentence."
```

Comment:
change attribute to `helpText` and set the value "Describe your altruistic community (DAC) in 1 sentence"